test/sequential/test-inspector-command-error.js

```diff
@@ -0,0 +1,12 @@
+// Flags: --expose-internals
+'use strict';
+
+process.config.variables.v8_enable_inspector = 1;
```

Comment:
Shouldn't the test just be skipped if inspector is disabled?

Reply:
Have handled it by calling `common.skipIfInspectorDisabled();`, it won't that be sufficient?

Reply:
Yes, my point is why is this line needed?

Reply:
I personally thought, we need to enable inspector (for every test). But re-reasoning now, I guess you are right, this might not be required and it contradicts with the call of `skipIfInspectorDisabled`. Will remove it.